### PR TITLE
fixes for Sql Server 2008 and early

### PR DESCRIPTION
**Pull request by darkdef**
- correct drop trigger function in function `tests\framework\db\mssql\ActiveRecordTest::testSaveWithComputedColumn`
- fix number of select query params for old sql server version, without correct offset support in function `framework\db\mssql\QueryBuilder::upsert`

| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Tests pass?   | ✔️/❌
| Fixed issues  |

### CODE REVIEW

**Comment by darkdef** [comment]
@bizley @samdark 
Please review fixes for SQL Server < 11

File: framework/db/mssql/QueryBuilder.php [hunk 1]
```diff
@@ -535,8 +538,18 @@ public function upsert($table, $insertColumns, $updateColumns, &$params)
         }
         $on = $this->buildCondition($onCondition, $params);
         list(, $placeholders, $values, $params) = $this->prepareInsertValues($table, $insertColumns, $params);
+
+        /**
+         * fix number of select query params for old sql server version, without correct offset support
```
Comment (samdark) [inline framework/db/mssql/QueryBuilder.php:543]:
```suggestion
         * Fix number of select query params for old MSSQL version that does not support offset correctly.
```

**Review by samdark — APPROVED** [review]
Need a changelog.

**Comment by samdark** [comment]
👍

**Pull request merged**